Misc/NEWS.d/next/Library/2018-03-11-08-44-12.bpo-33034.bpb23d.rst

```diff
@@ -0,0 +1 @@
+Providing an explicit message when casting a port that is a string instead of an integer.  Port could not be cast to integer value.
```

Comment:
Style nit: Please wrap lines at 80 characters.

Reply:
You may also want to mention `urlparse` in the news entry :) For context, this text is going to be listed at https://docs.python.org/3/whatsnew/changelog.html

Reply:
And please add "Patch by Matt Eaton."

Reply:
Thank you very much!